src/main/java/ToDo.java

```diff
@@ -0,0 +1,12 @@
+public class ToDo extends Task{
+
+    public ToDo(String name) {
+        super(name);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+    
```

Comment:
Spacing; new line/line break used here is different from above, please adhere to some form of consistency.